xds/internal/xdsclient/authority.go

```diff
@@ -246,6 +254,16 @@ func (a *authority) updateResourceStateAndScheduleCallbacks(rType xdsresource.Ty
 				continue
 			}
 
+			// Per A53, we want to delete the resource from cache based on the environment
+			// variable ignore_resource_deletion in server config.
```

Comment:
This comment could be made a little bit more useful I feel. Maybe something like:
```golang
// Per A53, resource deletions are ignored if the `ignore_resource_deletion` server feature is
// enabled through the bootstrap configuration. If the resource deletion is to be ignored,
// the resource is not removed from the cache and the corresponding 
// OnResourceDoesNotExist() callback is not invoked on the watchers.
```